src/components/SiteInfoInputForm.js

```diff
@@ -608,43 +610,41 @@ class SiteInfoInputForm extends Component {
                     </Text>
 
                     <Text style={labelStyle}>{title}</Text>
-                    <Input
+                    <TextInput
                         autoCapitalize={'words'}
-                        containerStyle={formInputStyle}
+                        style={[formInputStyle, textInputStyle]}
                         placeholder={add_site_title}
                         value={siteTitleText}
                         onChangeText={this.onUpdateSiteTitleText}
                     />
 
                     <Text style={labelStyle}>{description}</Text>
-                    <Input
-                        containerStyle={formInputStyle}
+                    <TextInput
+                        style={[formInputStyle, textInputStyle]}
                         placeholder={description_placeholder}
                         value={siteDescriptionText}
                         onChangeText={this.onUpdateSiteDescriptionText}
-                        inputContainerStyle={largeTextInput}
                         blurOnSubmit={true}
                         multiline={true}
                         autoGrow={true}
```

Comment:
In trying to figure out what our crash was, I refactored away from `react-native-elements` `<Input />` to the normal react-native `<TextInput />`. Nice benefit we got foe free is that the `autogrow` property is now respected, which checks off that long lasting bug.